addons/dragging/functions/fnc_handleAnimChanged.sqf

```diff
@@ -1,23 +1,21 @@
 #include "script_component.hpp"
 /*
  * Author: commy2
- * Handle the animaion for a Unit for Dragging Module
+ * Handle the animation for a unit for the dragging module.
  *
  * Arguments:
  * 0: Unit <OBJECT>
- * 1: animaion <STRING>
+ * 1: Animaion <STRING>
  *
  * Return Value:
  * None
  *
  * Example:
- * [_unit, "amovpercmstpsnonwnondnon"] call ace_dragging_fnc_handleAnimChanged;
+ * [player, "amovpercmstpsnonwnondnon"] call ace_dragging_fnc_handleAnimChanged;
  *
  * Public: No
 */
 
-//IGNORE_PRIVATE_WARNING ["_thisArgs", "_thisID"]; // From CBA_fnc_addBISEventHandler;
```

Comment:
sqf_linter will use these hints to reduce false positives
tool hasn't been updated in a long time so it's not that useful
but going forward there really isn't any reason to remove these lines

Reply:
Should I add them back?